SwiftUIDZ/SwiftUIDZ/SettingsView.swift

```diff
@@ -0,0 +1,19 @@
+// SettingsView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Экран настроек
+struct SettingsView: View {
+    // MARK: - Public properties
+
+    var body: some View {
+        Text("SettingsView")
```

Comment:
```suggestion
        Text("SettingsView")
```
 а Чо не в константах?)